editor/editor_file_system.cpp

```diff
@@ -2439,6 +2444,10 @@ void EditorFileSystem::reimport_files(const Vector<String> &p_files) {
 		} else {
 			pr.step(reimport_files[i].path.get_file(), i);
 			_reimport_file(reimport_files[i].path);
+
+			// We need to increment the counter, maybe the next file is multithreaded
+			// and don't have the same importer.
```

Comment:
```suggestion
			// and doesn't have the same importer.
```
or "they don't" if referring to both files.

Reply:
Done!